cmd/id.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+	"github.com/trento-project/agent/internal/agent"
+)
+
+func NewIDCmd() *cobra.Command {
```

Comment:
Maybe not better `NewAgentIDCmd`?
To follow along the `GetAgetnID` function name